test/multiverse/suites/roda/roda_instrumentation_test.rb

```diff
@@ -113,7 +113,7 @@ def test_transaction_name_error
       result = NewRelic::Agent::Instrumentation::Roda::TransactionNamer.transaction_name({})
 
       assert_equal NewRelic::Agent::UNKNOWN_METRIC, result
-      assert_logged(/NoMethodError.*Error encountered trying to identify Roda transaction name/)
+      assert_logged(/NoMethodError.*Error encountered trying to identify Roda transaction name/m)
```

Comment:
Bizarrely, we end up with a newline in the middle of the log message only with Ruby 3.1.4 (not with Ruby 2.7, 3.0, 3.2, etc.), so let's have `.*` match the newline.